presto-docs/src/main/sphinx/functions/khyperloglog.rst

```diff
@@ -0,0 +1,75 @@
+======================
+KHyperLogLog Functions
+======================
+
+Presto implements the `KHyperLogLog <https://research.google/pubs/pub47664/>`_
+algorithm and data structure. KHyperLogLog data structure can be created
+through :func:`khyperloglog_agg` function.
```

Comment:
nit: remove `function`. Usually we just reference the function with the link.

Reply:
Do you mean just surrounding it with double backquotes?

Reply:
I mean removing the word function at the end of the sentence.
```
... through :func:`khyperloglog_agg`.
```